client/state/plugins/premium/selectors.js

```diff
@@ -34,6 +39,7 @@ export const getPluginsForSite = function( state, siteId, whitelist = false ) {
 	}
 
 	return filter( pluginList, plugin => {
+		// eslint-disable-next-line no-extra-boolean-cast
```

Comment:
Wouldn't removing the `!!` work? :slightly_smiling_face: (That's the whole point of this linting rule -- the question I'm essentially asking is what makes this case different from the broader class of cases this lint is about?)

Reply:
It could work, sure. I actually did that initially, but then changed it back. There's at least one location where this whitelist is actually set from a query string parameter. We can't be sure what is set there unless we do a full audit. In that case, I could be introducing a change of the current behavior without knowing so. So I prefer going with adding an eslint disable rule for now, and addressing this manually with a proper audit and testing. This PR is already big enough, and introducing subtle changes that could cause subtle bugs is what I'd prefer to avoid. Hope that's alright!